api/binance/futures_client.py

```
@@ -1,901 +1,901 @@
+                try:
+                    response = requests.get(
+                        f"https://api.binance.com/api/v3/ticker/price?symbol={symbol}",
+                        timeout=2
+                    )
+                    if response.status_code == 200:
+                        data = response.json()
+                        current_price = float(data['price'])
+                        logger.info(f"Public API 가격: {symbol} = ${current_price:,.2f}")
+                    else:
+                        current_price = 50000.0
+                except:
```

Ruff (0.14.2)

Do not use bare `except`

(E722)